debug.js

```diff
@@ -49,8 +49,7 @@ export function debug(...args) {
 }
 
 chrome.storage.onChanged.addListener((changes, areaName) => {
-  let val = changes["debug-mode"]?.newValue
-  if (val !== undefined) {
-    debugMode = val
+  if ("debug-mode" in changes) {
```

Comment:
see #1 – should this be `changes.hasOwnProperty("debug-mode")`?

Reply:
AIUI it only matters if there are properties inherited through the prototype chain, which in this case I doubt there are (and even if they are, we'd want to pick up on it - no idea how Chrome implements this internally)

Reply:
Chrome's docs use `for in` without `hasOwnProperty`, which leads me to think this _should_ be fine...? https://developer.chrome.com/extensions/storage#examples

Happy to change if you feel particularly strongly though

Reply:
nah, is fine